R_Intro_Lesson.md

```diff
@@ -13,30 +9,22 @@ pre-installed RStudio on the workshop image. Start up a Jetstream
 m1.medium or larger using the ANGUS 2019 image [as per Jetstream startup
 instructions](jetstream/boot.html). Connect to your instance.
 
-We will need a password to connect to RStudio. We can set the password
-to our instance by running the following commands in bash (the password
-will not be visible on the screen):
-
-``` bash
-sudo passwd $USER
-```
-
-We will also need our instance username to connect to RStudio. To
+We will need our instance username to connect to RStudio. To
 determine what your username is, run:
 
 ``` bash
 echo My username is $USER
 ```
 
-Lastly, find the RStudio server interface Web address (by default in
+Find the RStudio server interface Web address (by default in
 port 8787) by running the following command:
 
 ``` bash
-echo http://(hostname):8787/
+echo http://$(hostname):8787/
```

Comment:
Can we add a brief description of the hostname?

Reply:
so i started trying to do this (describe what the host name is), and as usual it seemed to get complicated pretty quickly, ha. So i changed it so that we are just using the IP address, which works the same and we've already been using. 

In case you meant a description of what the command `hostname` is doing in that `echo` command, i added a note box reminding them of what's going on there :)

Reply:
Looks absolutely great! Thanks @AstrobioMike